scripts/selenium-test.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+setup_service xvfb
+setup_service google-chrome-stable 83.0.4103.61-1
```

Comment:
Use latest chrome available - `setup_service google-chrome-stable 87.0.4280.88-1`

Reply:
@vijetmahabaleshwar-okta 
That is done!